### PR TITLE
ecoinvent 3.10 compatibility

### PR DESCRIPTION
**Pull request by mrvisscher**
Replaced the default install chain with a `ProjectSetupWizard` that will either install the biosphere3 packaged with `bw2io`, or use `ecoinvent_interface` to download and install a specific version of ecoinvent and matching biosphere.

- [x] See where tests break
## Fixes
- [x] Fix default install tests
- [x] Check or remove Python 3.8 & 3.9 compatibility
- [x] Discuss Python 3.8 & 3.9 compatibility

## Checklist

- [ ] Keep pull requests small so they can be easily reviewed.
- [ ] Update the documentation, please follow the [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] Update tests.
- [ ] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `feature`, `ui`, `change`, `documentation`, `breaking`, `ci`
      as they show up in the changelog.
- [ ] Link this PR to related issues by using [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
- [ ] Add a milestone to the PR (and related issues, if any) for the intended release.
- [ ] Request a review from another developer.

### CODE REVIEW

File: activity_browser/mod/ecoinvent_interface/__init__.py
Comment (marc-vdm) [inline activity_browser/mod/ecoinvent_interface/__init__.py:1]: if this file is empty, can't we just remove it?

File: activity_browser/mod/ecoinvent_interface/release.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            with open(out_filepath, "rb") as source, open(
+                    directory / filename, "w", encoding="utf-8"
+            ) as target:
+                gzip_fd = gzip.GzipFile(fileobj=source)
+                target.write(gzip_fd.read().decode("utf-8-sig"))
+            try:
+                out_filepath.unlink()
+            except PermissionError:
+                # Error on Windows during testing
+                message = """"Can't automatically delete {out_filepath}
```
Comment (marc-vdm) [inline activity_browser/mod/ecoinvent_interface/release.py:65]: Same as earlier, `"` and `f""`

**Review by marc-vdm — COMMENTED** [review]
Minor comments, just see per comment if you change or reject, then looks fine to merge to me!

File: activity_browser/mod/ecoinvent_interface/__init__.py
Comment (mrvisscher) [reply activity_browser/mod/ecoinvent_interface/__init__.py:1]: No it's necessary to mark it as a package

**Pull request merged**